server/plugin.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/mattermost/mattermost-server/model"
 	"github.com/mattermost/mattermost-server/plugin"
 
-	zd "github.com/mattermost/mattermost-plugin-zoom/server/zoom"
+	zd "github.com/mattermost/mattermost-plugin-zoom/server/server/zoom"
```

Comment:
??? that seems a weird import path. What happened?

Reply:
From this branch? (I haven't merged Hanzei's changes were just added)

Reply:
@levb ah, because Hanzei's changes also changed the package name. The package name was causing that weird import path.